### PR TITLE
feat: new voting card

### PR DESCRIPTION
**Pull request by eddieferrer**
New voting card component with story

Screenshot 2024-10-08 at 1 11 46 PM
Screenshot 2024-10-08 at 1 11 29 PM

### CODE REVIEW

File: @kiva/kv-components/vue/KvVotingCardV2.vue [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	computed: {
+		cssProps() {
+			return {
+				'--background-image': `url(${this.backgroundImageUrl})`,
+			};
+		},
+	},
+	methods: {
+		castVote() {
+			this.$emit('vote');
```
Comment (mcstover) [inline @kiva/kv-components/vue/KvVotingCardV2.vue:83]: Don't we need to return or declare the emits?

Comment (eddieferrer) [reply @kiva/kv-components/vue/KvVotingCardV2.vue:83]: I think this works with the options API, changed it to use setup(), added emit test in story.

**Review by mcstover — APPROVED** [review]

File: @kiva/kv-components/vue/KvVotingCardV2.vue [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+		'vote',
+	],
+	setup(props, { emit }) {
+		const {
+			backgroundImageUrl,
+		} = toRefs(props);
+
+		const cssProps = computed(() => {
+			return {
+				'--background-image': `url(${backgroundImageUrl.value})`,
```
Comment (mcstover) [inline @kiva/kv-components/vue/KvVotingCardV2.vue:88]: Do we need to namespace this css variable so it's doesn't conflict with others on the page? Or will it be scoped into the component instance somehow?

Comment (eddieferrer) [reply @kiva/kv-components/vue/KvVotingCardV2.vue:88]: Maybe scoped? I'm not sure, no harm in namespacing it, thats a good idea

**Review by mcstover — APPROVED** [review]

**Pull request merged**